src/App.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from "react";
 import { getAllData } from "./util/index";
```

Comment:
I believe we do not need to import getAllData anymore as well as useState and useEffect ( for now ), but in regards to your code: It looks great!! Good job, Doug!